### DISCUSSION TITLE
GitHub Education Re-Verification process has broken redirection (impossible to apply)

### DISCUSSION THREAD

**Discussion by sronald01** [community/community#86486] [GitHub Education]
### Hi everyone,

Hi there,

I've read multiple recent reports here and can conclude GitHub made some changes to Educational benefits system, which may have limited valid submissions from being processed without any public notice. In the same time, it seems like they introduced some bugs to the system. Here's what I experience:
* On December 18th 2023 I received an email, asking to provide academic re-verification documents (on occasional basis)
* The link in the email (https://education.github.com/benefits) leads to valid page, and contains link to verification process inside (https://education.github.com/discount_requests/application)
* By following re-verification link (https://education.github.com/discount_requests/application), I personally get wrong redirect to GitHub Pro payment page: https://github.com/pricing

What I've tried:
* Cleared Cache & Cookies, re-logged-in. The broken redirection still persists.
* Contacted GitHub support (Ticket #2489085), but it got closed prematurely without any human response twice after several days. I re-opened it after it was closed automatically by responding to it, but now after it got closed again it does not allow any responses. The ticket describes broken redirection and I asked for their assistance on the case I describe here publicly now.
* Contacted GitHub billing support in regards to additional email I got from GitHub Billing offering me 100% discount for GitHub Pro 30-Days Access (why?), which I have not requested and that's why I opened another ticket to inquire about this weird promotional offer. I never used GitHub pro and did not request any coupons. The ticket (#2515381) got Archived without any human response after several days of inactivity.

Does anybody has a clue on how to proceed with it now since GitHub support is not responsive in a human way?

Hope these details help understand what's going on to everyone who may be stuck in the same situation now.

<details><summary>The emails I got:</summary>

![Screenshot 2024-01-10 at 11 11 01@2x](https://github.com/community/community/assets/129984590/31272c49-3a7c-4f49-a7ee-2691734134ca)
![Screenshot 2024-01-10 at 11 28 09@2x](https://github.com/community/community/assets/129984590/f2b959e8-4d32-47e3-bd72-126ec6f20a5a)
</details>

**Comment by 5k5000** [comment]
The same issue

**Comment by liuliangsir** [comment]
Same issue here. But I think the issue above might be a feature not a bug since most people have not been reporting that issue for a long time.

**Comment by CarmJos** [comment]
Any updates please?

**Comment by falahhasan** [comment]
Any updates please?

**Comment by xiaoxiaojiujiu** [comment]
same issue

**Comment by ExAxoncel** [comment]
Same issue here, and a lot same issues in this forum, but Github does nothing..

**Comment by huskyhao** [comment]
same issue

**Comment by gpzlx1** [comment]
Same issue. Very disappointed.

**Comment by puppetsheep** [comment]
Me too

**Comment by SakuraPuare** [comment]
same issue, how can we in touch with this problem

**Comment by Lyuansama** [comment]
Same issue

**Comment by irloc** [comment]
same issue, how can we in touch with this problem

**Comment by Kobe972** [comment]
Same issue

**Comment by ElenjaM** [comment]
I got the last e-mail too.
The whole problem is described in discussion: Third Rejection of my Request for Githup Pro as a student #130721
I thought it was my fault but obviously there are a lot of people with similiar problems.
Maybe it will help you too, if somebody is anwering my questions.

**Comment by evnusi37** [comment]
The same issue

**Comment by ISeeCan** [comment]
Same

**Comment by maddox05** [comment]
dude github sucks

**Comment by zhanghulu2004** [comment]
The same issue
